Package.swift

```diff
@@ -33,7 +33,7 @@ let package = Package(
           targets: ["Snapshotting"]),
     ],
     dependencies: [
-      .package(url: "https://github.com/EmergeTools/FlyingFox.git", exact: "1.0.0"),
```

Comment:
Can we pin to an exact version? We try to do that for all deps so nothing unexpected changes